docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -81,6 +81,10 @@ See [Troubleshooting](doc:troubleshooting#how-to-change-your-cache-directory) fo
 Recommended commands
 --------------------
 
+> 🚧 Autofixing goals
+>
+> The goals `fmt` and `fix` will attempt to automatically correct your code and then return zero if they were able to do so.  This generally counts as "success" by most CI systems.  Prefer the `lint` goal to fail the CI job on formatting or linting problems.
+
```

Comment:
I think it would be helpful to mention `lint` here as the checking version of `fmt`/`fix` suitable to use in CI.